added allowance cap explanation (#153)
decentraland/documentation: content/player/blockchain-integration/buying-nfts.md

Users can easily buy Land, Estates, Names, Wearables and Emotes using the [Marketplace](https://market.decentraland.org/). As easy as it seems to do it on the website, various Smart Contracts on the Ethereum Network and the Polygon Network make this possible.

> Lands, Estates and Names are commercialized in the marketplace using a different contract than the ones used for Wearables and Emotes, and Wearables that you buy from the store listed as primary sales use a different one as well. Users are also able to place bids for different NFTs which are also handled by different smart contracts.

Reviewer: ```suggestion
Lands, Estates, and Names are commercialized in the marketplace using a different contract than the ones used for Wearables and Emotes, and Wearables that you buy from the store listed as primary sales use a different one as well. Users can also place bids for different NFTs, which are also handled by other smart contracts.
```

As you can see, there are contracts both in the Ethereum and the Polygon networks. This is because different kinds of assets can be found in one network and the rest in the other. For example, LANDs, Estates, and Names are all in the Ethereum Network, while Wearables and Emotes are in the Polygon Network.

> **Networks are not interoperable, meaning that you cannot buy Wearables in the Ethereum network or place bids for LANDs in Polygon. Contracts can only interact with the assets found in their same networks. But still, the user can get Wearables or Emotes while connected to the Ethereum network thanks to [Meta-Transactions](https://docs.decentraland.org/player/blockchain-integration/transactions-in-polygon/), this is automatic in the Marketplace for a better UX**

Reviewer 2: Changes:
- `The user` for `users`, as there's no specific user we're talking about.
- Added Polygon to make it a bit clearer.
- Changed a bit the ending because it sounded better.
```suggestion
**Networks are not interoperable, meaning that you cannot buy Wearables in the Ethereum network or place bids for LANDs in Polygon. Contracts can only interact with the assets found in their same networks. But still, users can get Wearables or Emotes in Polygon while connected to the Ethereum network thanks to [Meta-Transactions](https://docs.decentraland.org/player/blockchain-integration/transactions-in-polygon/) which are automatically used in the Marketplace for a better UX**
```

## Marketplace

> The Marketplace Smart Contract is the one that allows users to create an Order to sell their assets.

Reviewer 2: If this is a technical document (specially because there's solidity code here), I wouldn't remove this explanation as it explains what can be done in each network. We can enhance it though:
```suggestion
The Marketplace Smart Contract in Ethereum allows users to create an Order to sell LANDs, Estates, Names and Legacy Wearables, while the one in Polygon only allows the creation of Orders for Wearables and Emotes.
```

)
```

Once an Order has been created on-chain, the asset will be listed for sale so that any user can buy it. 

> When a user wants to buy an asset listed in the Marketplace, they have to allow the Marketplace Smart Contract to operate with their MANA first. This is done by signing a transaction in MetaMask or the user's Wallet. Users can set an allowance for the amount of the asset they want to buy or a higher cap to continue using the Marketplace without changing it every time. We recommend using the Allowance cap by default.

Reviewer 2: Following that this is a technical document, we should either pause at:
```
When a user wants to buy an asset listed in the Marketplace, they have to allow the Marketplace Smart Contract to operate with their MANA first.
```
To include the signature of the transaction that is done against the MANA contract to change the allowance or have a section to describe how the allowance is performed.

Reviewer 3: ```suggestion
When a user wants to buy an asset listed in the Marketplace, they have to allow the Marketplace Smart Contract to operate with their MANA first. This is done by sending a transaction. Users can set an allowance for the amount of the asset they want to buy or a higher cap to continue using the Marketplace without changing it every time. We recommend using the Allowance cap by default.
```

> When a user wants to buy an asset listed in the Marketplace, they have to allow the Marketplace Smart Contract to operate with their MANA first. This is done by signing a transaction in MetaMask or the user's Wallet. Users can set an allowance for the amount of the asset they want to buy, or set a higher cap to continue using the Marketplace without changing the allowance every time. We recommend to use the Allowance cap by default.

Reviewer: ```suggestion
When a user wants to buy an asset listed in the Marketplace, they have to allow the Marketplace Smart Contract to operate with their MANA first. This is done by signing a transaction in MetaMask or the user's Wallet. Users can set an allowance for the amount of the asset they want to buy or a higher cap to continue using the Marketplace without changing it every time. We recommend using the Allowance cap by default.
```

> Any user with enough MANA, and with a not expired Order, can execute it and purchase the NFT if the value of the bought asset is lower than the allowance cap. An order is executed by calling the `executeOrder` function in the Marketplace Smart Contract.

Reviewer 3: ```suggestion
Any user with enough MANA can purchase the NFT if the value of the asset to be bought is lower than the allowance cap. An order is executed by calling the `executeOrder` function in the Marketplace Smart Contract.
```

> Any user with sufficient MANA, and as long as the Order is not expired, can execute the Order and purchase the NFT if the value of the asset bought is lower than the allowance cap. An order is executed by calling the `executeOrder` function in the Marketplace Smart Contract.

Reviewer: ```suggestion
Any user with enough MANA, and with a not expired Order, can execute it and purchase the NFT if the value of the bought asset is lower than the allowance cap. An order is executed by calling the `executeOrder` function in the Marketplace Smart Contract.
```
